bgp_config_test.go

```diff
@@ -0,0 +1,53 @@
+package packngo
+
+import (
+	"testing"
+)
+
+var configID string
+var configRequest CreateBGPConfigRequest
+
+func TestAccCreateBGPConfig(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, projectID, _ := setupWithProject(t)
+
+	configRequest = CreateBGPConfigRequest{
+		DeploymentType: "local",
+		Asn:            65000,
+		Md5:            "c3RhY2twb2ludDIwMTgK",
+	}
+
+	_, err := c.BGPConfig.Create(projectID, configRequest)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestAccGetBgpConfig(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c := setup(t)
+
+	config, _, err := c.BGPConfig.Get(projectID)
```

Comment:
again, I think that all the API resources (and ID variables) must be local to the test function. it's a bad idea to reuse projectID set from another function.